intellij-java-google-style.xml

```diff
@@ -306,16 +306,18 @@
         <option name="LABEL_INDENT_ABSOLUTE" value="false" />
         <option name="USE_RELATIVE_INDENTS" value="false" />
     </ADDITIONAL_INDENT_OPTIONS>
-    <ADDITIONAL_INDENT_OPTIONS fileType="java">
+	<codeStyleSettings language="JAVA">
+    <indentOptions>
         <option name="INDENT_SIZE" value="2" />
         <option name="CONTINUATION_INDENT_SIZE" value="4" />
-        <option name="TAB_SIZE" value="8" />
+        <option name="TAB_SIZE" value="2" />
         <option name="USE_TAB_CHARACTER" value="false" />
         <option name="SMART_TABS" value="false" />
         <option name="LABEL_INDENT_SIZE" value="0" />
         <option name="LABEL_INDENT_ABSOLUTE" value="false" />
         <option name="USE_RELATIVE_INDENTS" value="false" />
-    </ADDITIONAL_INDENT_OPTIONS>
+    </indentOptions>
+  </codeStyleSettings>
     <ADDITIONAL_INDENT_OPTIONS fileType="js">
```

Comment:
Are the other indent option tags incorrect as well?

Reply:
I'm not sure. I don't really have a way to test those so I was leaving it as is.